### ISSUE TITLE
Position of header element changes when navigating to Bootcamp link

### ISSUE THREAD

**Issue by prasadhonrao** [prasadhonrao/devcamper#88]
When navigating to the Bootcamp link from other sections of the site, the position of the header element changes unexpectedly. This should not happen, and the header should remain in the same position across all pages.

**Comment by AkankshShigwan** [comment]
- When navigate to bootcamps page from any another page header position getting change -> was happening because loader taking more space => resolved
- new issue found -> when you navigate from page which have scroll bar to page which don't have scroll bar the header fluctuates => my suggestion was to remove scroll bar and add down arrow (@prasadhonrao what do you think....)

**Comment by prasadhonrao** [comment]
Good analysis @AkankshShigwan. 

We should assume that some of these pages will have vertical scroll bar as content will flow from top to bottom, even for bootcamp details page. Eventually we will have pagination implemented at the bottom of the bootcamps page which will help us to limit number of bootcamps we display on home page.

We should not have a down arrow to navigate as that is not a standard practice in web.

**Comment by AkankshShigwan** [comment]
@prasadhonrao, ok so I have then hidden the scroll bar (page which scroll only scroll bar is hidden) to resolve these issues is it correct ?

**Comment by prasadhonrao** [comment]
@AkankshShigwan Not really. If the page contains any data (for e.g. footer) which is not visible to the user, then scroll bar is the indication that there is some additional data and user can scroll to it, so we should not hide the scroll bar. 

If you visit websites like facebook and twitter login page, they also show the scroll bar even though there is no additional content at the bottom of the page so you have that much width already allocated for scroll bar. Additionally they also show sticky header which makes it easy to navigate through different pages. 

Can you analyze something on those lines pls?

**Comment by AkankshShigwan** [comment]
@prasadhonrao in the evening meeting we will discuss this clearly as I have many questions.

**Comment by prasadhonrao** [comment]
Closed by PR 90
